.gitmodules

```diff
@@ -84,3 +81,7 @@
 [submodule "external/ffmpeg"]
 	path = external/ffmpeg
 	url = https://github.com/Vita3K/ffmpeg-core.git
+[submodule "external/boost"]
+	path = external/boost
+	url = https://github.com/totlmstr/ext-boost.git
```

Comment:
Let's fork this!

Reply:
Note that the branch being checked out there is `boost-trim`. The branch name is not seen here, [but they are indeed very different](https://github.com/totlmstr/ext-boost/compare/509da24bc9cf868c9029297c5b54426e7ede33ae...c9ee09ddddab89c7f35d1f6d9f493d846c1f39cd).